packages/armbian/builddeb

```diff
@@ -249,7 +249,7 @@ cp "$($MAKE -s -f $srctree/Makefile image_name)" "$tmpdir/$installed_image_path"
 if is_enabled CONFIG_OF_EARLY_FLATTREE; then
 	# Only some architectures with OF support have this target
 	if [ -d "${srctree}/arch/$SRCARCH/boot/dts" ]; then
-		$MAKE -f $srctree/Makefile INSTALL_DTBS_PATH="$tmpdir/usr/lib/$packagename" dtbs_install
+		$MAKE -f $srctree/Makefile INSTALL_DTBS_PATH="$tmpdir/usr/lib/linux-image-$version" dtbs_install
```

Comment:
So _that's_ where it was hiding! 👏 
And this ends up in `image` package, not `dtb` package, so this solves 2 (or more) problems!

Reply:
Yes, here.
But it's not that simple.
Scripts from u-boot-menu packages search for files in the path:
`/usr/lib/linux-image-$version/device-tree/`
And in our packages it is:
`/usr/lib/linux-image-$version/$vendor/` for sunxi `/usr/lib/linux-image-$version/allwinner/`

Reply:
Hmm. Indeed. If `/usr/lib/$packagename` (old path) is not required by anything, we might just change to the scheme required by Debian tools. That would squash 3 problems: 1) `flash-kernel`, 2) `u-boot-menu`, 3) the fact that the `-image` package will contain the DTBs. Old stuff using `u-boot` directly will require the `-dtb` package, otherwise no. Looks like a 4x win for me.